test/wallet/createVirtualIban.js

```diff
@@ -0,0 +1,85 @@
+const { expect } = require('chai');
+const Chance = require('chance');
+const moment = require('moment');
+
+const Lemonway = require('../../');
+
+const chance = new Chance();
+
+describe('createVirtualIban', function () {
+  this.timeout(2000000);
```

Comment:
Why ?

Reply:
All tests have this 🙈 Really don't know why 🤔 @jguepin any idea ?

Reply:
Lemon Way sandbox used to be slow to answer calls. We can try to remove/reduce it.